cloudbuild.yaml

```diff
@@ -0,0 +1,17 @@
+timeout: 3000s
+steps:
+  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20211013-1be7868d8b
+    entrypoint: make
+    args:
+      - image-build
+      - image-push
+    env:
+    - IMAGE_REGISTRY=gcr.io/$PROJECT_ID
+    - _GIT_TAG=$_GIT_TAG
+    - IMAGE_EXTRA_TAG_NAMES=$_PULL_BASE_REF
+substitutions:
+  _GIT_TAG: '0.0.0'
```

Comment:
does this get automatically substituted based on the github tag?

Reply:
yup, is a place holder

Reply:
see other projects just put 12345 
e.g 

https://github.com/kubernetes-sigs/cluster-api/blob/main/cloudbuild.yaml#L29